src/webcomponents/commons/filters/go-accessions-filter.js

```diff
@@ -43,12 +41,12 @@ export default class GoAccessionsFilter extends LitElement {
         };
     }
 
-    _init() {
+    #init() {
         this._prefix = "gaf-" + UtilsNew.randomString(6) + "_";
         this.selectedTerms = "";
         this.ontologyTerm = "GO";
-        this.ontologyFilter = "go";
-        this._config = {...this.getDefaultConfig(), ...this.config};
+        this.ontologyFilter = "GO";
```

Comment:
As I see neither `this.ontologyTerm` and `this.ontologyFilter` are in use in the class. I would suggest removing them.